src/cli/utils.ts

```diff
@@ -341,3 +345,36 @@ export function getTagsFromOptions(
 ): { name: string; value: string }[] {
   return parseTags(options.tags);
 }
+
+export function currencyFromOptions<
+  O extends GlobalOptions & { currency?: string },
```

Comment:
NIT: O and 0 look very similar in monospaced fonts. I was incorrectly reading the type parameter as ZERO rather than the letter O here. Consider using a more familiar type variable like `T`.